<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeaderFramerForm</class>
 <widget class="QWidget" name="HeaderFramerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="le_header">
       <property name="placeholderText">
        <string>Header (e.g. 0x80ff, 0b011101, XYZ)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_byteAlign">
       <property name="text">
        <string>Byte-aligned</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_fixedLength">
       <property name="text">
        <string>Fixed Frame Length:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_frameLength"/>
     </item>
     <item>
      <widget class="QCheckBox" name="cb_prePad">
       <property name="text">
        <string>Pre-pad:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="sb_prePad"/>
     </item>
     <item>
      <widget class="QPushButton" name="pb_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tw_headers">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pb_remove">
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
